### PR TITLE
[refactor](executor)Add % suffix for cpu_hard_limit

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

Add % suffix for cpu_hard_limit.

```
create workload group if not exists test1
properties (
    "cpu_share"="10",
    "memory_limit"="1%",
    "enable_memory_overcommit"="true",
    "cpu_hard_limit"="1%"
);

| 10085 | test1  | enable_memory_overcommit | true       |
| 10085 | test1  | cpu_hard_limit           | 1%         |
| 10085 | test1  | max_concurrency          | 2147483647 |
| 10085 | test1  | queue_timeout            | 0          |
| 10085 | test1  | max_queue_size           | 0          |
| 10085 | test1  | memory_limit             | 1%         |
| 10085 | test1  | cpu_share                | 10         |

```

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroup.java [hunk 1]
```diff
@@ -103,7 +103,12 @@ private WorkloadGroup(long id, String name, Map<String, String> properties, long
             properties.put(ENABLE_MEMORY_OVERCOMMIT, properties.get(ENABLE_MEMORY_OVERCOMMIT).toLowerCase());
         }
         if (properties.containsKey(CPU_HARD_LIMIT)) {
-            this.cpuHardLimit = Integer.parseInt(properties.get(CPU_HARD_LIMIT));
+            String cpuHardLimitStr = properties.get(CPU_HARD_LIMIT);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroup.java:106]: if user write code like cpu_hard_limit="1%    ", then the code is wrong. I think you should trim

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by wm1581066 — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**